chain/actors/builtin/miner/miner.go

```diff
@@ -10,6 +10,7 @@ import (
 	actorstypes "github.com/filecoin-project/go-state-types/actors"
 	"github.com/filecoin-project/go-state-types/big"
 	minertypes13 "github.com/filecoin-project/go-state-types/builtin/v13/miner"
+	minertypes16 "github.com/filecoin-project/go-state-types/builtin/v16/miner"
 	minertypes "github.com/filecoin-project/go-state-types/builtin/v9/miner"
```

Comment:
Can we unify these to v16?

Reply:
Yes, but it's costly: https://github.com/filecoin-project/lotus/pull/12960

Reply:
Commented there.